frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-exclusions-section-card/feature-flag-exclusions-section-card.component.html

```diff
@@ -10,7 +10,7 @@
   <!-- header-right -->
   <app-common-section-card-action-buttons
     header-right
-    [showPrimaryButton]="true"
+    [showPrimaryButton]="(permissions$ | async)?.featureFlags.update"
     [primaryButtonText]="'feature-flags.details.add-exclude-list.button.text' | translate"
```

Comment:
add disable check to `[actionsDisabled]` in `app-feature-flag-exclusions-table` below.

Reply:
Good catch! Thank you!